library/src/main/java/com/android/volley/Request.java

```diff
@@ -54,6 +56,7 @@
      * Identifies if the request is finished
      */
     private boolean mFinished;
+    private int mStatus = -1;
```

Comment:
What is this status? Is it an html status code? If so can we name it such? Also, comments please.